src/fsharp/FSharp.Core.nuget/FSharp.Core.4.1.xxx.nuspec

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <!-- rename to FSharp.Core when done -->
+        <id>FSharp.Core</id>
+        <description>
+            FSharp.Core redistributables from Visual F# 4.1.0
+                Supported Platforms:
+                    .NET 2.0                    (net20)
+                    .NET 4.0                    (net40)
+                    .NET 4.5+                   (net45)
+                    .NET Portable Profile 7     (portable-net45+netcore45)
+                    .NET Portable Profile 47    (portable-net45+sl5+netcore45)
+                    .NET Portable Profile 78    (portable-net45+netcore45+wp8)
+                    .NET Portable Profile 259   (portable-net45+netcore45+wpa81+wp8)
+                    .NET Standard 1.6           (netstandard1.6)
+                    MonoAndroid, MonoTouch, Xamarin.iOS (portable-net45+monoandroid10+monotouch10+xamarinios10)
+                    XamarinMac mobile           (xamarinmac20)
+        </description>
+        <language>en-US</language>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <version>$version$</version>
+        <authors>$authors$</authors>
+        <licenseUrl>$licenseUrl$</licenseUrl>
+        <projectUrl>$projectUrl$</projectUrl>
+        <tags>$tags$</tags>
+        <dependencies>
+            <group>
+                <dependency id="System.ValueTuple" version="4.3.0" />
+            </group>
+            <group>
```

Comment:
this is missing framework restriction, like `<group targetFramework=".NETStandard1.6">`